src/posthog-core.ts

```diff
@@ -313,7 +316,8 @@ export class PostHog {
     private _internalEventEmitter = new SimpleEventEmitter()
 
     // Legacy property to support existing usage - this isn't technically correct but it's what it has always been - a proxy for flags being loaded
-    public get decideEndpointWasHit(): boolean {
+    // TODO DYLAN: do I need to keep this, or can I rename it?
+    public get flagsEndpointWasHit(): boolean {
```

Comment:
its a public API so I'd say no, what you can do is to add a @deprecated annotation, add a new method, and this legacy method just forwards the call to the new method